nasa/earthdata-search
# EDSC-3317: Deprecating m url param

# Overview

### What is the feature?

Deprecates the m param

### What is the Solution?

When a user uses the m param, the application will decode the parameter into their new replacement parameters. The user will also see the deprecated parameter modal.

### What areas of the application does this impact?

/search, /project

# Testing

### Reproduction steps

- **Environment for testing:** any
- **Collection to test with:** any

1. Use any `m` parameter
2. Confirm any non-default or non-preference values are translated to their new counterparts
3. Confirm the deprecated parameter modal is displayed
4. Confirm the `m` parameter is listed as deprecated in the modal

# Checklist
